components/script/dom/xrframe.rs

```diff
@@ -52,16 +54,17 @@ impl XRFrameMethods for XRFrame {
     }
 
     /// https://immersive-web.github.io/webxr/#dom-xrframe-getviewerpose
-    fn GetViewerPose(&self, reference: Option<&XRReferenceSpace>) -> Option<DomRoot<XRViewerPose>> {
-        // We assume the reference space is eye level for now
-        // since it's the only one 3DOF devices support
-        if reference.is_some() {
-            // it's not possible to obtain a reference
-            // space at all yet
-            return None;
+    fn GetViewerPose(&self, reference: &XRReferenceSpace) -> Option<DomRoot<XRViewerPose>> {
+        if let Some(_) = reference.downcast::<XRStationaryReferenceSpace>() {
+            // For 3DOF devices all three kinds of reference spaces are identical
+            // XXXManishearth support originOffset
```

Comment:
If we're checking this in, it should be a TODO preferably with a link to an issue.